src/components/tooltip/Tooltip.tsx

```diff
@@ -87,6 +87,8 @@ export type TooltipProps = {
     children: React.ReactChild;
     /** A CSS class for the tooltip */
     className?: string;
+    /** Should the tooltip show again after user closes it once it is eligible again. Defaults to `false` */
+    shouldTooltipShowAferUserClose?: boolean;
```

Comment:
it looks like this was copied with typo: should be shouldTooltipShowAfterUserClose

Reply:
good catch, updated